tensorboard/webapp/metrics/views/card_renderer/scalar_card_types.ts

```diff
@@ -100,8 +100,8 @@ export enum ColumnHeaderType {
 
 export interface ColumnHeader {
   type: ColumnHeaderType;
-  name: string;
-  displayName: string;
+  name?: string; // TODO: make required after internal changes are make
+  displayName?: string; // TODO: make required after internal changes are make
```

Comment:
nit: Can you add names after TODO?
TODO(jameshollyer): Makes required... made

Reply:
Done.